jujugui/static/gui/src/app/app.js

```diff
@@ -491,7 +491,7 @@ YUI.add('juju-gui', function(Y) {
       let password = null;
       let macaroons = null;
       let credentials = this.user.controller;
-      if (credentials.areAvailable) {
+      if (credentials && credentials.areAvailable) {
```

Comment:
This is the case where someone loads the gui server without using guiproxy so they're not connected to a controller. We just need these sections to not error so that they get give the notice about using guiproxy.